### DISCUSSION TITLE
Error: hostname is not valid

### DISCUSSION THREAD

**Discussion by serdiyon** [elementary/installer#643] [Q&A]
⚠️ I Need Help With This please...
Installing elementary os 6 odin failed,possibly due to a hardware error,the device may not restart properly.
Details:
get block size    "sys/class/block/sdd"
Error:error while installing system hostname is not valid.

**Comment by cassidyjames** [comment]
@serdiyon what is the name of the install ISO you're using? This issue _should_ be fixed by https://github.com/elementary/installer/pull/604 in the`elementaryos-6.0-stable.20210831.iso` image. 

If you're still using `elementaryos-6.0-stable.20210810.iso` or possibly the `elementaryos-6.0-stable.20210823.iso` file, you should re-download it from the homepage.
